components/email-mgt/org.wso2.carbon.email.mgt/src/main/java/org/wso2/carbon/email/mgt/EmailTemplateManagerImpl.java

```diff
@@ -274,37 +257,39 @@ private String getDefaultNotificationLocale(String notificationChannel) {
 
     @Override
     public NotificationTemplate getNotificationTemplate(String notificationChannel, String templateType, String locale,
-                                                        String tenantDomain) throws NotificationTemplateManagerException {
+                                                        String tenantDomain)
+            throws NotificationTemplateManagerException {
 
-        return getNotificationTemplate(notificationChannel, templateType, locale, tenantDomain, null);
+        return getNotificationTemplate(notificationChannel, templateType, locale, tenantDomain, null, true);
```

Comment:
shall we comment the reason to set resolve=true in org level email template get method

Reply:
Addressed with https://github.com/wso2-extensions/identity-event-handler-notification/pull/291/commits/e631725df3f22df5aad8e687f6a8d483171c8b2b